src/mode/modeHandler.ts

```diff
@@ -280,10 +295,11 @@ export class SearchState {
     }
   }
 
-  constructor(direction: SearchDirection, startPosition: Position, searchString = "") {
+  constructor(direction: SearchDirection, startPosition: Position, searchString = "", { isRegex = false } = {}) {
```

Comment:
(Which will destructure the object immediately and assign isRegex = false if it can't find a value)